release.sh

```diff
@@ -30,4 +30,16 @@ git push origin $version
 # Finally, push docker images
 docker tag $user/$image:latest $user/$image:$version
 docker push $user/$image:$version
-docker push $user/$image:latest
\ No newline at end of file
+docker push $user/$image:latest
+
+#Push docker images to GCR
+export CLOUD_SDK_REPO="cloud-sdk-$(lsb_release -c -s)"
+echo "deb http://packages.cloud.google.com/apt $CLOUD_SDK_REPO main" | sudo tee -a /etc/apt/sources.list.d/google-cloud-sdk.list
+curl https://packages.cloud.google.com/apt/doc/apt-key.gpg | sudo apt-key add -
+sudo apt-get update && sudo apt-get install google-cloud-sdk -y
+gcloud auth activate-service-account --key-file=${HOME}/gcloud-service-key.json
```

Comment:
Can we use the built-in support instead of installing this from the release script?
https://circleci.com/docs/2.0/google-container-engine/#authenticate-to-googles-container-registry

Reply:
Tried that before, 
 - image: google/cloud-sdk
is not compatible with https://github.com/gochain-io/explorer/blob/master/.circleci/config.yml#L4

Reply:
Yeah, you have to use `docker` instead. I switched over gochain a while back: https://github.com/gochain-io/gochain/blob/master/.circleci/config.yml

Everything is faster in general, and it would allow us to skip the 'install docker' step too.